<ui version="4.0" >
 <class>LowPanel</class>
 <widget class="QWidget" name="LowPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>671</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>25</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Brightness</string>
      </property>
      <widget class="QSlider" name="brightnessSlider" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdBrightness" >
       <property name="geometry" >
        <rect>
         <x>130</x>
         <y>20</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Contrast</string>
      </property>
      <widget class="QSlider" name="contrastSlider" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdContrast" >
       <property name="geometry" >
        <rect>
         <x>130</x>
         <y>20</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Threshold</string>
      </property>
      <widget class="QSlider" name="thresholdSlider" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="value" >
        <number>50</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdThreshold" >
       <property name="geometry" >
        <rect>
         <x>130</x>
         <y>20</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0" >
        <number>50</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
